Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py

```diff
@@ -16,6 +16,7 @@
 
```

Comment:
Disregard changes to GSB. I'm going to revert these since they have not been tested.